platform/security/security-migratable/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -14,23 +14,9 @@
            xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
            xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
 
-    <bean id="migratableUtil" class="org.codice.ddf.migration.util.MigratableUtil"/>
-
     <bean id="securityMigratable"
-          class="org.codice.ddf.security.migratable.impl.SecurityMigratable">
-        <argument ref="securityDescribable"/>
-        <argument ref="migratableUtil"/>
-    </bean>
-
-    <bean id="securityDescribable"
-          class="org.codice.ddf.migration.DescribableBean">
-        <argument value="1.0"/>
-        <argument value="ddf.security"/>
-        <argument value="Security Migration"/>
-        <argument value="Exports Security system files"/>
-        <argument value="Codice"/>
-    </bean>
+          class="org.codice.ddf.security.migratable.impl.SecurityMigratable"/>
 
     <service id="securityMigratableService" ref="securityMigratable"
-             interface="org.codice.ddf.migration.ConfigurationMigratable"/>
+             interface="org.codice.ddf.migration.Migratable" ranking="900"/>
```

Comment:
❓ Arbitrary, or are you actually using service rankings to order migratables? They're independent of each other, aren't they?

Reply:
Ranking is being used and supported. The fact is that the Security migratable and the platform migratable and not quite independent and we have been talking about combining them. When you see the profile migratable appearing, you will notice its ranking to be much higher so it runs first.